Sprint-2/debug/author.js

```diff
@@ -11,6 +11,6 @@ const author = {
   alive: true,
 };
 
-for (const value of author) {
-  console.log(value);
+for (let key in author) {
+  console.log(author[key]);
```

Comment:
Hi, thanks for your question.
 `for...of` does not work directly with objects because it is designed to iterate over iterable objects like arrays or strings, not plain objects. Since objects are not iterable by default, you can use `for...in` to loop through the object's keys or `Object.values()`/`Object.keys()` to get an iterable array of values or keys. For example:

```javascript
for (const value of Object.values(author)) {
  console.log(value);
}